### PR TITLE
Add basic support for ZigDC

### PR DESCRIPTION
**Pull request by xyzroe**
Add basic support for ZigDC - Zigbee 6ch DC power meter

### CODE REVIEW

File: src/devices/xyzroe.ts [hunk 1]
```diff
@@ -54,6 +67,22 @@ function getSortedList(source: { [key: string]: number }): string[] {
     return result;
 }
 
+const getKey = (object: { [key: string]: number }, value: number) => {
```
Comment (Koenkk) [inline src/devices/xyzroe.ts:70]: Use `utils.getKey` instead

File: src/devices/xyzroe.ts [hunk 2]
```diff
@@ -33,6 +33,19 @@ const bindCommandList = {
     'recall_scene_5': 0x0B,
 };
 
+const switchTypesList = {
+    'switch': 0x00,
+    'single click': 0x01,
```
Comment (Koenkk) [inline src/devices/xyzroe.ts:38]:
```suggestion
    'single_click': 0x01,
```

File: src/devices/xyzroe.ts [hunk 3]
```diff
@@ -33,6 +33,19 @@ const bindCommandList = {
...
+    'multi-click': 0x02,
```
Comment (Koenkk) [inline src/devices/xyzroe.ts:39]:
```suggestion
    'multi_click': 0x02,
```

File: src/devices/xyzroe.ts [hunk 4]
```diff
@@ -33,6 +33,19 @@ const bindCommandList = {
...
+    'reset to defaults': 0xff,
```
Comment (Koenkk) [inline src/devices/xyzroe.ts:40]:
```suggestion
    'reset_to_defaults': 0xff,
```

**Comment by xyzroe** [comment]
It seems like everything has been fixed. Thank you!

**Comment by Koenkk** [comment]
Thanks!

**Pull request merged**